src/olympia/bandwagon/tests/test_indexers.py

```diff
@@ -0,0 +1,74 @@
+# -*- coding: utf-8 -*-
+from olympia.amo.tests import TestCase
+from olympia.bandwagon.tasks import attach_translations
+from olympia.bandwagon.indexers import CollectionIndexer
+from olympia.bandwagon.models import Collection
+from olympia.constants.applications import FIREFOX
+
+
+class TestCollectionIndexer(TestCase):
+    def setUp(self):
+        super(TestCollectionIndexer, self).setUp()
+        self.attrs = ('author_username', 'created', 'id', 'listed', 'modified',
+                      'monthly_subscribers', 'rating', 'slug', 'subscribers',
+                      'type', 'weekly_subscribers')
+        self.collection = Collection.objects.create(
+            application=FIREFOX.id, monthly_subscribers=666, rating=4,
+            subscribers=999, weekly_subscribers=133,)
+        self.indexer = CollectionIndexer()
+
+    def _extract(self):
+        qs = Collection.objects.filter(id__in=[self.collection.pk])
+        for t in (attach_translations,):
+            qs = qs.transform(t)
+        self.collection = list(qs)[0]
+        return self.indexer.extract_document(self.collection)
+
+    def test_extract_attributes(self):
+        extracted = self._extract()
+        for attr in self.attrs:
+            assert extracted[attr] == getattr(self.collection, attr)
```

Comment:
Same here, can we turn this around and check that everything in `extracted` is in indeed in `self.attrs`? Or implement other sanity checks as suggested above.